client/src/pages/SearchMovie/index.js

```diff
@@ -1,42 +1,73 @@
-import React, { useState } from 'react'
+import React, { useState, useCallback, useMemo } from 'react'
 import debounce from 'lodash.debounce'
 
-import { SearchMovieContainer } from './styled'
+import { SearchMovieContainer, ClearImg, Button } from './styled'
 import { BebasText } from '../../styles/Text'
 import { SearchBar, SearchInput, SearchImg } from '../../styles/SearchBar'
 import { useConfiguration } from '../../providers/Configuration'
 import { authFetch } from '../../auth'
 import Results from './Results'
+import { useMovies } from '../../providers/Movies'
 
 const SearchMovie = () => {
   const { apiUrl } = useConfiguration()
+  const { movies } = useMovies()
+  const [inputValue, setInputValue] = useState('')
   const [movieResults, setMovieResult] = useState([])
```

Comment:
Since this is the result of search, this should be renamed to `[searchResults, setSearchResults]`